.env.ci.example

```diff
@@ -11,7 +11,6 @@ MUTUALAID_DATABASE_PASSWORD=testing123
 
 # SendGrid api key
 SENDGRID_API_KEY=123
-# SendGrid credentials
```

Comment:
including this in this PR bc it's making it easier for me to cross-compare my local env files